### ISSUE TITLE
documentation apt install ubuntu 20 troubles

### ISSUE THREAD

**Issue by iranilom** [blockstack/stacks-blockchain#1855]
Sorry I'm not so familiary 

Someone can help me? 
What i'm doing wrong?

https://docs.blockstack.org/stacks-blockchain/install-api

sudo apt-get install -y python-pip memcached rng-tools python-dev libmemcached-dev zlib1g-dev libgmp-dev libffi-dev libssl-dev

Lendo listas de pacotes... Pronto
Construindo árvore de dependências       
Lendo informação de estado... Pronto
Note, a seleccionar 'python-dev-is-python2' em vez de 'python-dev'
E: Impossível encontrar o pacote python-pip

solved 

sudo apt search python-pip
Sorting... Pronto
Full Text Search... Pronto
python-pip-whl/focal,focal,now 20.0.2-5ubuntu1 all [installed]
  Python package installer

sudo apt-get install -y python-pip-whl

is it ok a lot of warnings...

sudo npm -g install aglio
npm WARN deprecated chokidar@1.7.0: Chokidar 2 will break on node v14+. Upgrade to chokidar 3 with 15x less dependencies.
npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm WARN deprecated jade@1.11.0: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated constantinople@3.0.2: Please update to at least constantinople 3.1.1
npm WARN deprecated transformers@2.1.0: Deprecated, use jstransformer
npm WARN deprecated request@2.81.0: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated hawk@3.1.3: This module moved to @hapi/hawk. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
npm WARN deprecated har-validator@4.2.1: this library is no longer supported
npm WARN deprecated sntp@1.0.9: This module moved to @hapi/sntp. Please make sure to switch over as this distribution is no longer supported and may contain bugs and critical security issues.
npm WARN deprecated cryptiles@2.0.5: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated boom@2.10.1: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated hoek@2.16.3: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
npm WARN deprecated mkdirp@0.3.5: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated json3@3.3.2: Please use the native JSON object instead of JSON 3
/usr/local/bin/aglio -> /usr/local/lib/node_modules/aglio/bin/aglio.js
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/aglio/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ aglio@2.3.0
updated 1 package in 8.149s

then 
pip install . ????? doesn't  work
pip install .
ERROR: Directory '.' is not installable. Neither 'setup.py' nor 'pyproject.toml' found.

then again 

pip install -r api/requirements.txt
ERROR: Could not open requirements file: [Errno 2] Arquivo ou diretório não encontrado: 'api/requirements.txt'

blockstack setup_wallet
blockstack: command not found
blockstack api start
blockstack: command not found

deactivate
ok

./build_docs.sh public_api
bash: ./build_docs.sh: File or directory not found

**Steps To Reproduce**
Please provide detailed instructions (e.g. command line invocation with parameters) to reproduce the behavior.

 - OS: Ubuntu 20.04.1 LTS

**Comment by diwakergupta** [comment]
@iranilom sorry, it's a bit hard to follow your issue. Please keep in mind Stacks 1.0 is legacy software that will be superceded by Stacks 2.0 (currently in development). Make sure to clone the `stacks-1.0` branch: https://github.com/blockstack/stacks-blockchain/tree/stacks-1.0

Could you specify exactly which step your install fails at? Looks like `aglio` installed, and then `pip install .` failed? Were you already inside the `blockstack-core` directory?

**Comment by iranilom** [comment]
Hi,
I understand, i'm a unexperient ...sorry, but i really enjoid participate
of this tests!
If you can tell me where can i found a tutorial, step by step, for initiate
again since stacks-1.0?
Understanting what you said i think maybe this being issue.
Thank you very much for your help!
Beste regards,
@iranilom - Molinari

Atenciosamente.

MARCO ANTONIO MOLINARI
11 97134 1004 - Vivo


Em ter., 8 de set. de 2020 às 13:49, Diwaker Gupta <notifications@github.com>
escreveu:

**Comment by diwakergupta** [comment]
@iranilom it sounds more like you need support: I suggest asking for help on our forums or on Discord. I'm closing this out as there's no bug.

**Comment by iranilom** [comment]
git clone https://github.com/blockstack/stacks-blockchain/tree/stacks-1.0
Cloning into 'stacks-1.0'...
fatal: repository '
https://github.com/blockstack/stacks-blockchain/tree/stacks-1.0/' not found



Atenciosamente.

MARCO ANTONIO MOLINARI
11 97134 1004 - Vivo


Em ter., 8 de set. de 2020 às 13:49, Diwaker Gupta <notifications@github.com>
escreveu:

**Comment by diwakergupta** [comment]
```bash
$ git clone https://github.com/blockstack/stacks-blockchain.git
$ git checkout -b stacks-1.0 origin/stacks-1.0
```

**Comment by iranilom** [comment]
Downloaded .zip
Installing now

Thanks

Atenciosamente.

MARCO ANTONIO MOLINARI
11 97134 1004 - Vivo


Em ter., 29 de set. de 2020 às 15:23, Diwaker Gupta <
notifications@github.com> escreveu:
